src/content/docs/guides/big-projects.mdx

```diff
@@ -95,34 +95,89 @@ Let's jump to `app/backend/biome.json`, where we need to enable the linting:
+Since v2, Biome supports monorepos out of the box, and you'll need to set up the project in the following way.
 
-In order to have the best developer experience despite the current limitation, it's advised to have a `biome.json` at the root of the monorepo, and use the [`overrides`](/reference/configuration/#overrides) configuration to change the behaviour of Biome in certain packages.
+<Steps>
+1. Create a `biome.json` file at the root of the monorepo. We will use the recommended rules, and customise the formatter
+    options.:
 
-In the following example we disable the rule `suspicious/noConsoleLog` inside the package `packages/logger`.
-
-```json title="biome.jsonc"
-
-{
-  "linter": {
-    "enabled": true,
-    "rules": {
-      "recommended": true
+    ```json title="biome.json"
+    {
+        "linter": {
+            "enabled": true,
+            "rules": {
+                "recommended": true
+            }
+        },
+        "formatter": {
+            "lineWidth": 120,
+            "indentStyle": "space",
+            "indentWidth": 6,
+        }
     }
-  },
-  "overrides": [{
-      "include": ["packages/logger/**"],
-      "linter": {
-        "rules": {
-          "suspicious": {
-            "noConsoleLog": "off"
-          }
+    ```
+    This file is called **root configuration** and it sets the base options inside the project. However, nested
+    configurations can decide to adhere to these options or not. Let's see how.
+
+2. Create nested configuration files, one in each package, where is needed. These nested configuration files must have the
+    field `"root"` set to `false`. Failing to do so will result into an error. Also, we want these packages the follow
+    the formatting standards set up in the root configuration. In order to so, we will use a **new microsyntax**
+    available in Biome v2, which is `"extends": "//"`. This syntax tells Biome to extends from the **root configuration**,
+    regardless of where the nested configuration is.
+
+    Let's create two configuration files,
+    one inside `packages/logger` and one inside `packages/generate`. In the former we will disable `noConsole`, and in
+    `packages/generate` we will disable the formatter because those are files that are code-generated:
+
+    ```json title="packages/logger/biome.json"
+    {
+        "root": false,
+        "extends": "//",
+        "linter": {
+            "rules": {
+                "suspicious": {
+                    "noConsole": "off"
+                }
+            }
         }
-      }
-  }]
-}
-```
+    }
+    ```
+
+    ```json title="packages/generate/biome.json"
+    {
+        "root": false,
+        "extends": "//",
+        "formatter": {
+            "enabled": false
+        }
+    }
+    ```
+    However, when you use the microsyntax `extends: "//"`, you can **omit `"root": true`, because it implies that this configuration isn't a
+    a root configuration, and it wants extends from the root configuration:
```

Comment:
nit: "microsyntax" is jargon
```suggestion
    However, when you use `extends: "//"`, you can **omit `"root": true`, because it implies that this configuration isn't a
    a root configuration, and it wants extends from the root configuration:
```